hcvswitch.sh

```diff
@@ -74,7 +74,7 @@ hcv_use() {
 hcv_conf() {
     local KEY="$1"
     VAL=$(grep -e "$KEY" "$HCVSWITCH_CURRENT" | cut -f "2-" -d ':' | sed -e 's! !!g; s!\"!!g')
-    echo "$VAL"
+    echo "${VAL:=$HCVSWITCH_DEFAULT_USER}"
```

Comment:
Into this concept. WIth this configuration it will use `HCVSWITCH_DEFAULT_USER` as the default for _any_ config key tho. The substitution should probably happen around line [85](https://github.com/otakup0pe/hcvswitch/blob/eda1bb4f6ef9f9ae8ab7f825ee6ac5a2abad6612/hcvswitch.sh#L85). It should also fail gracefully if the default is not set and the README shoudl be updated. <3